src/auth/screens/events.screen.js

```diff
@@ -149,7 +149,7 @@ class Events extends Component {
       case 'MemberEvent':
         return `${userEvent.payload.action}`;
       case 'PublicEvent':
-        return translate('auth.events.issuesEvent', language);
+        return translate('auth.events.publicEvent', language);
```

Comment:
I think this will turn out to look like `User made public {{repo}}`. If at all possible, I think it would be best to word it `User made {{repo}} public`.

![screen shot 2017-08-12 at 9 51 05 am](https://user-images.githubusercontent.com/10191084/29242519-d3c6910a-7f43-11e7-87ce-a3246b3fff4f.png)